operator/api/vectorized/v1alpha1/cluster_types.go

```diff
@@ -1210,36 +1216,20 @@ func (s *ClusterStatus) SetRestarting(restarting bool) {
 	s.DeprecatedUpgrading = restarting
 }
 
-// GetCurrentReplicas returns the current number of replicas that the controller wants to run.
-// It returns 1 when not initialized (as fresh clusters start from 1 replica)
-func (r *Cluster) GetCurrentReplicas() int32 {
-	if r == nil {
+func (r *Cluster) GetReplicas() int32 {
```

Comment:
nit: comment for this public method is missing

Reply:
done